sdks/java/core/src/test/java/org/apache/beam/sdk/util/GcsUtilTest.java

```diff
@@ -321,28 +321,74 @@ public void testAccessDeniedObjectThrowsIOException() throws IOException {
 
   @Test
   public void testGetSizeBytes() throws Exception {
-    GcsOptions pipelineOptions = gcsOptionsWithTestCredential();
-    GcsUtil gcsUtil = pipelineOptions.getGcsUtil();
-
-    Storage mockStorage = Mockito.mock(Storage.class);
-    gcsUtil.setStorageClient(mockStorage);
-
-    Storage.Objects mockStorageObjects = Mockito.mock(Storage.Objects.class);
-    Storage.Objects.Get mockStorageGet = Mockito.mock(Storage.Objects.Get.class);
-
-    when(mockStorage.objects()).thenReturn(mockStorageObjects);
-    when(mockStorageObjects.get("testbucket", "testobject")).thenReturn(mockStorageGet);
-    when(mockStorageGet.execute()).thenReturn(
-        new StorageObject().setSize(BigInteger.valueOf(1000)));
-
-    assertEquals(1000, gcsUtil.fileSize(GcsPath.fromComponents("testbucket", "testobject")));
+    String singleFile = "--batch_foobarbaz\n"
+        + "Content-Type: application/http\n"
+        + "\n"
+        + "HTTP/1.1 200 OK\n"
+        + "Content-Length: 708\n"
+        + "\n"
+        + "{\n"
```

Comment:
Does there exist a Java API to create an object that will dump this JSON, so you don't have to know and maintain the exact format?